docs/compliance/nist/800_53.md

```diff
@@ -1,9 +1,9 @@
 ---
 description: NIST 800-53 compliance guidance for Docker Enterprise Edition
 keywords: standards, compliance, security
-title: NIST 800-53
+title: NIST SP 800-53
 ---
 
-NIST Special Publication 800-53 contains a listing of security controls that entities often reference for ensuring the security and validity of their technology systems. The official listing of controls can be found on the NIST website [here](https://nvd.nist.gov/800-53).
+NIST Special Publication (SP) 800-53 contains a listing of security controls that entities often reference for ensuring the security and validity of their technology systems. The official listing of controls can be found on the NIST website [here](https://nvd.nist.gov/800-53).
```

Comment:
"entities"?

"NIST website" should be the link text.

Reply:
Updating